PeakOperation/Core/RetryingOperation.swift

```diff
@@ -17,7 +17,7 @@ open class RetryingOperation<Output>: ConcurrentOperation, ProducesResult {
     /// This is checked, and if it is of type `failure(...)`, then the `retryStrategy` will be executed.
     public var output: Result<Output, Error> = Result { throw ResultError.noResult }
 
-    var failureCount = 0
+    public var failureCount = 0
```

Comment:
If this needs to be `public`, it should probably be `private(set)`.

Reply:
Good point. Done